css/styles.css

```
@@ -48,18 +50,72 @@ header {
 .sounds__item {
   margin: 0 24px 72px;
   height: 110px;
+  padding-top: 20px;
   width: 84px;
+  cursor:pointer;
+  opacity: 0.6;
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
+}
+
+.sounds__item-active {
```

I wonder if this more closely aligns with BEM standards:
```suggestion
.sounds__item--active {
```